tests/filesystem/test_filesystem.py

```diff
@@ -18,22 +18,32 @@
     assert_query_data,
     TEST_STORAGE_ROOT,
 )
+from tests.filesystem.utils import unpack_factory_args
 
-from .settings import GLOB_RESULTS, TESTS_BUCKET_URLS
+from .settings import GLOB_RESULTS, FACTORY_ARGS
 
 
-@pytest.mark.parametrize("bucket_url", TESTS_BUCKET_URLS)
+@pytest.mark.parametrize("factory_args", FACTORY_ARGS)
```

Comment:
this generates test names that are not readable. ie `:test_file_list[glob_params0-factory_args4]`. Could you please provide the names with `ids` (it is a pytest trick)

Reply:
Oh cool - those test names were making me cry!
Is this it?  https://hackebrot.github.io/pytest-tricks/param_id_func/

Reply:
Done